js.go

```diff
@@ -860,6 +864,14 @@ func RateLimit(n uint64) SubOpt {
 	})
 }
 
+// Stream binds a consumer to a stream explicitly based on a name.
+func Stream(name string) SubOpt {
```

Comment:
BindStream() to be more explicit about what we want the system to do?

Reply:
Updated to `BindStream()`